public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -347,7 +351,7 @@ export const AnomalyHistory = (props: AnomalyHistoryProps) => {
           props.isHistorical,
           taskId.current
         );
-        return dispatch(searchResults(entityResultQuery));
+        return dispatch(searchResults(entityResultQuery, resultIndex));
```

Comment:
most of the queries / results parsing has been changed as part of multi-category filtering PR. Will need to keep an eye on that when merging.